internal/cmd/db_create.go

```diff
@@ -165,6 +165,17 @@ var createCmd = &cobra.Command{
 			extensions = "all"
 		}
 
+		var timestamp *time.Time
+		if fromDBFlag != "" {
+			ts, err := time.Parse("2006-01-02T15:04:05", timestampFlag)
```

Comment:
Is timestamp flag required for forking? I thought it's also possible to just for current db, no?